### PR TITLE
GTL going live

### PR DESCRIPTION
**Pull request by valoumiaou**
# DE PR

## Describe your changes

Please include a summary of the changes:

This PR [adds/removes/fixes/replaces] the [feature/bug/etc]. 

Tag a reviewer if necessacy  @github/username 

## Jira ticket number and/or notion link

JIRA-ticket_number

### Type of change
- [ ] Fix (non-breaking change which corrects expected behavior)
- [ ] New fields (non-breaking change)
- [x] New table (non-breaking change)
- [x] Concept change (potentially breaking change which modifies fields according to new or evolving business concepts) 
- [ ] Table deletion (potentially breaking change which adds functionality/ table)
      
### Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] My code passes CI/CD tests
- [ ] I updated README.md
- [x] I have updated the dag
- [ ] If my changes concern incremental table, I have altered their schema to accomodate with field's creation/deletion
- [ ] I have made corresponding changes to the [tables documentation](https://www.notion.so/passcultureapp/Documentation-Tables-175a397a8e854ff4a55ae4f3620dbe3b)
- [ ] I have made corresponding changes to the [fields glossary](https://www.notion.so/passcultureapp/854a436a8f1541e1b6ec2a65f8bab600?v=798024ba90404b139e5a17407a3bc604)
- [x] I will create a review on slack and ensure to specify the duration of the review task: short (<10min), medium (<30min), long (>30min)

### Added tests?
- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help
- [ ] ⏰ no, but I created a ticket

### CODE REVIEW

File: orchestration/dags/dependencies/import_analytics/sql/clean/applicative_database/titelive_gtl_mapping.sql [hunk 1]
```diff
@@ -0,0 +1,12 @@
+SELECT
```
Comment (LucileRainteau) [inline orchestration/dags/dependencies/import_analytics/sql/clean/applicative_database/titelive_gtl_mapping.sql:1]: on mettrait pas ça dans dbt intermediate ?

File: orchestration/dags/dependencies/import_analytics/sql/analytics/enriched_offer_metadata.sql [hunk 2]
```diff
@@ -110,31 +110,40 @@ offer_metadata_id AS (
...
         case 
-            when enriched_items.category_id like "LIVRE" then gtl.gtl_label_level_3 
+            when enriched_items.category_id like "LIVRE" then gtl_book.gtl_label_level_3
+            when enriched_items.category_id like "MUSIC" then gtl_music.gtl_label_level_3
             else null 
         end as gtl_label_level_3,
         case 
-            when enriched_items.category_id like "LIVRE" then gtl.gtl_label_level_4 
+            when enriched_items.category_id like "LIVRE" then gtl_book.gtl_label_level_4
+            when enriched_items.category_id like "MUSIC" then gtl_music.gtl_label_level_4
```
Comment (cdelabre) [inline orchestration/dags/dependencies/import_analytics/sql/analytics/enriched_offer_metadata.sql:136]: Français x Anglais ? LIVRE / MUSIC ?

**Review by cdelabre — COMMENTED** [review]
☁️

File: orchestration/dags/dependencies/import_analytics/sql/analytics/enriched_offer_metadata.sql (hunk 2 above)
Comment (cdelabre) [reply orchestration/dags/dependencies/import_analytics/sql/analytics/enriched_offer_metadata.sql:136]: Book non ? https://github.com/pass-culture/data-gcp/pull/2368/files#diff-33d4bd45f6bbaa9069a95d855359b92d480daba18af66925057bba0968f3f4a3R45

**Pull request merged**